boards/silabs/dev_kits/xg24_dk2601b/xg24_dk2601b.yaml

```diff
@@ -1,5 +1,5 @@
-identifier: efr32xg24_dk2601b
-name: BRD4601B
+identifier: xg24_dk2601b
+name: xG24 Dev Kit (BRD2601B)
```

Comment:
Sometime `name` provides the PN (brdXXXXX), sometime the OPN (SLTBXXX, xGXX-XXXX, etc..). What is the best policy? Should we mention both? I am in favor of mention the PN somewhere since it is the most reliable information when you find a random board on your desk.